tests/resources/conformance_suites/dba/fr/fr22-testcase.xml

```diff
@@ -18,6 +18,7 @@
             <instance readMeFirst="true">fr22-invalid-negative.xbrl</instance>
         </data>
         <result>
+            <error>DBA.FR21</error>
```

Comment:
Is there a challenge in getting these docs not to fire DBA.FR21? I'm fine with this but just wanting to confirm. If it's as easy as changing a couple of fact values or adding a fact then I think it's worth doing for this to be a clean testcase, otherwise it's fine to keep as-is.

Reply:
There is.  Because the fr22-invalid-negative and fr23-invalid-negative test cases have negative values for Assets and Liabilities   Resultantly when the comparison is made between the two values they aren't equal.  Making them both negative. would fix the fr21 error but then both the fr22-invalid-negative and fr23-invalid-negative would fire fr22 and fr23 errors.